.kitchen.cloud.yml

```diff
@@ -9,7 +9,7 @@ driver_config:
   openstack_tenant: <%= ENV['OS_TENANT_NAME'] %>
   floating_ip: <%= ENV['OS_FLOATING_IP'] %>
   floating_ip_pool: <%= ENV['OS_FLOATING_IP_POOL'] %>
-  flavor_ref: <%= ENV['OS_FLAVOR_REF'] %>
+  flavor_ref: 'm1.medium'
```

Comment:
Its a pretty standard flavor name and I know it works with that exact one. I'm fine making it an env var, I just felt that it would be more complicated to tell everyone that this one needs to be different.